subscription-engine/kafka-ws/README.md

```diff
@@ -316,7 +343,25 @@ If you are developing, you can use `nodemon` too:
 ```shell
 npm run dev
 ```
+__NOTE THAT_ in order to use WebSocket with nginx, Kong, API gateway or similar, look for timeout
```

Comment:
I would put the **N**ginx software with the first letter capitalized just like **K**ong, but it is just a suggestion.